src/main/java/org/logstash/dissect/DelimOne.java

```diff
@@ -0,0 +1,40 @@
+package org.logstash.dissect;
+
+class DelimOne implements Delim {
+    private final byte b;
```

Comment:
what about `delimiterCharacter` or the like as field name? `b` is kinda generic. :)